protocol/scripts/deploys/sepolia/v2/contract-addresses.ts

```diff
@@ -18,181 +18,192 @@ import {
-    CORE: {
-        TEMPLE_TOKEN: string,
-        CIRCUIT_BREAKER_PROXY: string,
-        // GNOSIS_SAFE_GUARD: string,
-        EXECUTOR_MSIG: string,
-        RESCUER_MSIG: string,
-    },
-    TREASURY_RESERVES_VAULT: {
-        ADDRESS: string,
-        D_USD_TOKEN: string,
-        D_TEMPLE_TOKEN: string,
-        TPI_ORACLE: string,
-    },
-    TEMPLE_LINE_OF_CREDIT: {
-        ADDRESS: string,
-        CIRCUIT_BREAKERS: {
-            DAI: string,
-            TEMPLE: string,
-        },
-        INTEREST_RATE_MODELS: {
-            LINEAR_WITH_KINK: string,
-        },
-    },
-    RAMOS: {
-        AUTOMATION_EOA: string,
-        TEMPLE_DAI: {
-            ADDRESS: string,
-            POOL_HELPER: string,
-            AURA_STAKING: string,
-            FEE_COLLECTOR: string,
-            BPT_TOKEN: string,
-            POOL_ID: string,
-        },
-    }
-    STRATEGIES: {
-        DSR_BASE_STRATEGY: {
-            ADDRESS: string,
-            EXECUTOR_MSIG: string,
-            RESCUER_MSIG: string,
-            // No circuit breakers for DSR base strategy
-        }
-        TEMPLE_BASE_STRATEGY: {
-            ADDRESS: string,
-            EXECUTOR_MSIG: string,
-            RESCUER_MSIG: string,
-            // No circuit breakers for Temple base strategy
-        },
-        GNOSIS_SAFE_STRATEGY1: {
-            ADDRESS: string,
-            EXECUTOR_MSIG: string,
-            RESCUER_MSIG: string,
-            UNDERLYING_GNOSIS_SAFE: string,
-            CIRCUIT_BREAKERS: {
-                DAI: string,
-                TEMPLE: string,
-            },
-        },
-        RAMOS_STRATEGY: {
-            ADDRESS: string,
-            EXECUTOR_MSIG: string,
-            RESCUER_MSIG: string,
-            CIRCUIT_BREAKERS: {
-                DAI: string,
-                TEMPLE: string,
-            },
-        },
-        TLC_STRATEGY: {
-            ADDRESS: string,
-            EXECUTOR_MSIG: string,
-            RESCUER_MSIG: string
-        },
-    },
-    EXTERNAL: {
-        MAKER_DAO: {
-            DAI_TOKEN: string,
-            DAI_JOIN: string,
-            POT: string,
-        },
-        BALANCER: {
-            VAULT: string,
-            HELPERS: string,
-        },
-    }
+  CORE: {
+      TEMPLE_TOKEN: string,
+      CIRCUIT_BREAKER_PROXY: string,
+      // GNOSIS_SAFE_GUARD: string,
+      EXECUTOR_MSIG: string,
+      RESCUER_MSIG: string,
+  },
+  TREASURY_RESERVES_VAULT: {
+      ADDRESS: string,
+      D_USD_TOKEN: string,
+      D_TEMPLE_TOKEN: string,
+      TPI_ORACLE: string,
+  },
+  TEMPLE_LINE_OF_CREDIT: {
+      ADDRESS: string,
+      CIRCUIT_BREAKERS: {
+          DAI: string,
+          TEMPLE: string,
+      },
+      INTEREST_RATE_MODELS: {
+          LINEAR_WITH_KINK: string,
+      },
+  },
+  RAMOS: {
+      AUTOMATION_EOA: string,
+      TEMPLE_DAI: {
+          ADDRESS: string,
+          POOL_HELPER: string,
+          AURA_STAKING: string,
+          AURA_TOKEN: string,
```

Comment:
If we don't need these new params for testnet, then i think we can just leave out?